templates/content-product.php

```diff
@@ -20,7 +20,7 @@
 global $product;
 
 // Ensure visibility.
-if ( empty( $product ) || false === wc_get_loop_product_visibility( $product->get_id() ) || ! $product->is_visible() ) {
```

Comment:
We should change the `wc_get_loop_product_visibility()` function to make sure it doesn't runs `wc_set_loop_product_visibility()` anymore.
Note that some people may have replaced those templates in their themes.

Reply:
Not sure if I understand what you mean. `wc_get_loop_product_visibility` is just doing a call to `wc_get_loop_prop`, or do you mean something else?